crates/assistant/src/assistant_panel.rs

```diff
@@ -79,6 +80,34 @@ use workspace::{
 use workspace::{searchable::SearchableItemHandle, NewFile};
 use zed_actions::InlineAssist;
 
+pub struct NewContext;
+
+impl Action for NewContext {
```

Comment:
We'll want to use the `gpui::actions!` macro to define the `Action` instead of writing out an `impl` by hand.

Reply:
I implemented this -- but noticed the only place that macro is uses is in the `actions.rs` of `Editor`. What is the best place for this?

Reply:
It should probably go with the rest of the assistant actions:

https://github.com/zed-industries/zed/blob/89487772b021d31e49a608934bff226b631d5870/crates/assistant/src/assistant.rs#L53-L68

Reply:
Missed this, thank you!